pypasta/Repository/Mbox.py

```diff
@@ -392,6 +492,20 @@ def __init__(self, config):
         log.info('  ↪ loaded invalid mail index: found %d invalid mails'
                  % len(self.invalid))
 
+        # If Patchwork projects are defined in the config, no need to load raw mboxes and pubins.
+        self.patchwork_projects = []
+        if len(config.mbox_patchwork_projects):
+            log.info('Loading Patchwork projects...')
+            for project_id, f_mbox_raw, listaddr in config.mbox_patchwork_projects:
+                self.lists.add(listaddr)
+                f_index = os.path.join(
+                    config.d_mbox, 'index', 'patchwork.%u' % project_id)
+                project = PatchworkProject(
+                    project_id, self.d_mbox, f_index, f_mbox_raw, **config.patchwork)
+                for message_id in project.get_ids():
+                    self.add_mail_to_list(message_id, listaddr)
+                self.patchwork_projects.append(project)
+
```

Comment:
Hmm. I can see a lot of duplicated startup code for all three different mailboxes. Maybe we can somehow consolidate the startup. But let's do that later.

Reply:
Yes, that's a good idea